lib/LaTeXML/Core/Gullet.pm

```diff
@@ -535,30 +531,51 @@ sub readKeyword {
   return; }
 
 # Return a (balanced) sequence tokens until a match against one of the Tokens in @delims.
-# In list context, also returns the found delimiter.
+# Note that Braces on input hides the contents from matching,
+# so this assumes there wont be braces in $delim!
+# But, see readUntilBrace for that case.
+# NOTE: Add error handling, $token undef detection, etc!!!!!
```

Comment:
Nice `NOTE` :>

Reply:
yeah, except I actually did add the error handling, but forgot to remove the comment!